# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: .cargo/config.toml.in
# Commit: ad954710c53f
# Full Hash: ad954710c53fa84879169b4f3d7994bb5f2ab714
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2024-06-03 16:07:28
# Regressor Bug: 1890186
# File Overlap Count: 5
# Description:
#   Bug 1890186: Update cubeb-coreaudio-rs to 1796ace5bd. r=cubeb-reviewers,padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D212311
# ==============================================================================

diff -r 8952a815f4b4 -r ad954710c53f .cargo/config.toml.in
--- a/.cargo/config.toml.in	Mon Jun 03 13:31:38 2024 +0300
+++ b/.cargo/config.toml.in	Mon Jun 03 10:03:39 2024 +0000
@@ -70,9 +70,9 @@
 rev = "3495905752a4263827f5d43737f9ca3ed0243ce0"
 replace-with = "vendored-sources"
 
-[source."git+https://github.com/mozilla/cubeb-coreaudio-rs?rev=4cc24c7dc74a4801455ba0bbe20ea67a49f28514"]
+[source."git+https://github.com/mozilla/cubeb-coreaudio-rs?rev=1796ace5bdd08ec8baa56bbf7170a08d760c984b"]
 git = "https://github.com/mozilla/cubeb-coreaudio-rs"
-rev = "4cc24c7dc74a4801455ba0bbe20ea67a49f28514"
+rev = "1796ace5bdd08ec8baa56bbf7170a08d760c984b"
 replace-with = "vendored-sources"
 
 [source."git+https://github.com/mozilla/cubeb-pulse-rs?rev=8678dcab1c287de79c4c184ccc2e065bc62b70e2"]